packages/cli/src/commands/auth/login.ts

```diff
@@ -116,4 +128,29 @@ export default class Login extends Command {
   private get configuration() {
     return new Config(this.config.configDir, this.error);
   }
+
+  private async getAllOrgsUserHasAccessTo() {
+    const orgs = await (
+      await new AuthenticatedClient().getClient()
+    ).organization.list();
+    return (orgs as unknown) as OrganizationModel[];
```

Comment:
You're sure about this, it doesn't look like `list()` returns this kind of array, at least directly.
Without checking out your branch, and by checking https://github.com/coveo/platform-client/blob/aeda85c635bb13c5de97bfc08ac8f2e7f08e4c30/src/resources/BaseInterfaces.ts,
I think that the `OrganizationModel[]` is in `.list().items` imo

I'll check out the branch and check later.

Reply:
![image](https://user-images.githubusercontent.com/12366410/112356444-fe113d80-8ca4-11eb-92b3-664384a7bcb5.png)
Yep.

Reply:
Yeah, it's a bug in the platform client, if you ask for "all" orgs, non paginated, it returns a different type than if you ask for paginated call.

So in this case the typing is invalid.

It's something I had identified as a bugfix for platform client, but never got the time to submit a fix...

Yeah, my bad.

Reply:
@louis-bompart :

PR for platform-client here: https://github.com/coveo/platform-client/pull/274